server/conn.go

```diff
@@ -609,31 +611,41 @@ func (cc *clientConn) dispatch(data []byte) error {
 		if len(data) > 0 && data[len(data)-1] == 0 {
 			data = data[:len(data)-1]
 		}
+		cc.ctx.SetProcessInfo(hack.String(data), t, cmd)
```

Comment:
I think we should consider `ss.SecureText()`, as it does here:https://github.com/pingcap/tidb/pull/7844/files#diff-350127760839dbfd52d23927f7ff2d95L225